File: gMission/customCalloutView/Bubble.m
//
//  BusPointCell.m
//  ZNBC
//
//  Created by 杨晓龙 on 13-4-11.
//  Copyright (c) 2013年 yangxiaolong. All rights reserved.
//

#import "Bubble.h"

@implementation Bubble

- (id)initWithFrame:(CGRect)frame
{
    self = [super initWithFrame:frame];
    if (self) {
        NSLog(@"initWithFrame");
        self.layer.borderColor = [UIColor blueColor].CGColor;
        [[self naviButton] setBackgroundColor:[UIColor blueColor]];
        [[self naviButton] layer];
        self.naviButton.layer.cornerRadius = 10;
    }
    
    return self;
}


//- (void)dealloc {
//    [_aliasLabel release];
//    [_speedLabel release];
//    [_degreeLabel release];
//    [_nameLabel release];
//    [super dealloc];
//}
@end

File: gMission/Bubble.swift
//
//  Bubble.swift
//  gMission
//
//  Created by Joshua Zhao on 14-7-29.
//  Copyright (c) 2014年 Joshua Zhao. All rights reserved.
//

import Foundation
import UIKit

@objc(LocationBubble)
class LocationBubble:UIView{

    @IBOutlet var nameLabel:UILabel!
    @IBOutlet var locationLabel:UILabel!
    @IBOutlet var timeLabel:UILabel!
    @IBOutlet var askBtn:UIButton!
    @IBOutlet var naviBtn:UIButton!
    
    required init(coder: NSCoder) {
        fatalError("NSCoding not supported")
    }
    
    override init(frame: CGRect) {
        super.init(frame: frame)
        naviBtn.backgroundColor = UIColor.blueColor()
        naviBtn.layer.cornerRadius = 4
    }
    
    @IBAction func AskBtnClicked(){
    
    }
    
    @IBAction func NaviBtnClicked(){
        
    }
    
}